packages/form-core/tests/FieldApi.spec.ts

```diff
@@ -642,7 +642,7 @@ describe('field api', () => {
       return
     })
 
-    const mockOnChangeAsync = vi.fn().mockImplementation(async ({ value }) => {
```

Comment:
If `async` is necessary in places like these it might make sense to add `eslint-disable` to reduce noise in the output.